### DISCUSSION TITLE
`wagmi generate` not working on v2

### DISCUSSION THREAD

**Discussion by drake-nathan** [wevm/wagmi#3449] [Q&A]
I'm working on upgrading a nextjs app to wagmi/viem v2 using bun. I can't get `wagmi generate` to work. I can get it to work using `bunx @wagmi/cli generate`. But it won't work in my `predev` command anymore. Any ideas?

**Comment by tmm** [comment]
Is there an error message or what happens? Can use something like [asciinema.org](https://asciinema.org) to share a recording.
